### ISSUE TITLE
Mobile install not work [BROMITE]

### ISSUE THREAD

**Issue by Milor123** [xcanwin/KeepChatGPT#15]
Error:

```
Invalid script header. Invalid RunAtDeclaration
document-idie
```

![image](https://user-images.githubusercontent.com/14153649/232354535-34260836-bbc6-4823-8ac1-2caa0b282f78.png)

**Comment by xcanwin** [comment]
Fixed in version 7.8, thanks for reminding: [commit](https://github.com/xcanwin/KeepChatGPT/commit/d91a4be9dcc8d22fcdaa7b07609a68fab70a2fe1)

**Comment by Milor123** [comment]
> Fixed in version 7.8, thanks for reminding: [commit](https://github.com/xcanwin/KeepChatGPT/commit/d91a4be9dcc8d22fcdaa7b07609a68fab70a2fe1)

Thank you very much for your project! I am very happy!
